phpunit.xml.dist

```diff
@@ -30,4 +30,8 @@
         </whitelist>
     </filter>
 
+    <php>
+        <ini name="precision" value="8"/>
+    </php>
```

Comment:
I'm not saying you should not change the precision, I'm saying you should say why (and "why not?" is not a valid reason). When you say why, you should said it in the commit message. BTW, your commit should be split into two commits (it contains the "and" word), and the second commit message should look like this : 

```
provide php-ini setting for precision

Here you explain why you are doing that.
```

Why do the tests fail if you do not set this precision ?